gulpfile.js

```diff
@@ -201,6 +201,18 @@ const mappings = {
     'dir': 'mysqlManagement/lib',
     'source': 'arm-rdbms/2017-04-30-preview/swagger/Rdms.MySQL.json'
   },
+  'monitor': {
+    'dir': 'monitor/lib',
+    'source': 'monitor/compositeMonitorClient.json',
+    'ft': 1,
+    'modeler': 'CompositeSwagger'
+  },
+  'monitor.management': {
+    'dir': 'monitorManagement/lib',
+    'source': 'arm-monitor/compositeMonitorManagementClient.json',
+    'ft': 1,
+    'modeler': 'CompositeSwagger'
+  },
```

Comment:
I think this should have a corresponding deletion in "insights" and that was missed. I'm doing it now, but would be nice to get a confirmation